packages/devtools_app/lib/src/performance/flutter/performance_screen.dart

```diff
@@ -172,4 +243,25 @@ class _PerformanceScreenBodyState extends State<PerformanceScreenBody>
       recordedObject: 'CPU samples',
     );
   }
+
+  void _exportPerformance() {
+    final exportedFile = controller.exportData();
+
+    Notifications.of(context)
+        .push('Successfully exported $exportedFile to ~/Downloads directory');
+  }
+
+  @override
+  FutureOr<void> processOfflineData(CpuProfileData offlineData) async {
+    if (!offlineMode) await controller.clear();
```

Comment:
is the call to clear necessary? I'm confused why we would ever be calling `processOfflineData` unless we are in `offlineMode`?

Reply:
Yeah, I think the line is obsolete since this method is never called if it isn't offline mode. I should've realized to remove the lne instead of adding a boolean check.